watch_service.go

```diff
@@ -236,10 +262,7 @@ func (w *Watcher) Watch(listener WatchListener) error {
 	ch := make(chan *WatchResult, 32)
 	go w.notifier(listener, ch)
 
-	w.listenersMutex.Lock()
-	w.updateListenerChans = append(w.updateListenerChans, ch)
-	w.listenersMutex.Unlock()
-
+	// check latest and give to notifer (via channel) asap
```

Comment:
nit: check latest and give to notifer -> check the latest value and give it to the notifier?

Reply:
SGTM. Let me update the comment 👍

Reply:
PTAL @hyangtack 🙇